Hash_map/include/CGAL/Tools/chained_map.h

```diff
@@ -187,19 +165,17 @@ inline void chained_map<T, Allocator>::insert(std::size_t x, T y)
 template <typename T, typename Allocator>
 void chained_map<T, Allocator>::rehash()
 {
-  old_table = table;
-  old_table_end = table_end;
-  old_table_size = table_size;
-  old_table_size_1 = table_size_1;
-  old_free = free;
+  chained_map_elem<T>* old_table = table;
```

Comment:
I don't know if you want to simplify the typedefs:
```c++
typedef chained_map_elem<T>*  chained_map_item;
typedef chained_map_item item;
```
and then just use one of them consistently?